src/sidebar/CustomModelBox.tsx

```diff
@@ -69,6 +60,15 @@ export default function CustomModelBox({ enabled, encodedValues }: CustomModelBo
         editor.categories = categories
     }, [encodedValues])
 
+    const triggerRouting = useCallback(
+        (event: React.KeyboardEvent<HTMLInputElement>) => {
+            if (event.ctrlKey && event.key === 'Enter') {
+                dispatchCustomModel(editor.value, isValid, true)
+            }
+        },
+        [editor, isValid]
+    )
+
```

Comment:
Not sure if this is better or worse, but we can also use the event handling of the text component using this API: https://github.com/graphhopper/custom-model-editor/blob/693a44097271ed84f603f46efbfcfc56ab97b608/src/custom_model_editor.js#L84-L86

This is how we did it in GH maps 1.0 as well: https://github.com/graphhopper/graphhopper/blob/876ffaa9fb990e2ea812c8956091809084025dcd/web-bundle/src/main/resources/com/graphhopper/maps/js/main-template.js#L153

And is it possible to send the request/json without validation when pressing Ctrl+Enter? @karussell should we do this? This is what we did for the old maps client.

Reply:
> @karussell should we do this?

:+1:

Reply:
Using Ctrl+Enter we can now force the request regardless of the client side validation. https://github.com/graphhopper/graphhopper-maps/pull/183/commits/78c612cfea1c13ed0e319e9f64d70a885de88955